demo/deploy-mysql.sh

```diff
@@ -49,8 +49,8 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
       containers:
-      - image: mysql:5.6
-        name: mysql
+      - image:
+        name: mysql/mysql-server
```

Comment:
This is definitely me playing devil's advocate -
is it worth it to template it in case someone wants to swap it with something like MariaDB and play with it?
I would say that those people **could just find this line** and update it manually - but since we're reworking some of this already it's something to think about.